input/include/wiiu/input.h

```diff
@@ -33,15 +33,15 @@
 #include <wiiu/kpad.h>
 #include <wiiu/pad_strings.h>
 
-#include "../../input/input_driver.h"
-#include "../../input/common/hid/hid_device_driver.h"
-#include "../../tasks/tasks_internal.h"
-#include "../../input/connect/joypad_connection.h"
-#include "../../retroarch.h"
-#include "../../verbosity.h"
-#include "../../command.h"
-#include "../../gfx/video_driver.h"
-#include "wiiu_hid.h"
+#include <input/input_driver.h>
+#include "../../common/hid/hid_device_driver.h"
+#include <tasks/tasks_internal.h>
+#include <input/connect/joypad_connection.h>
+#include <retroarch.h>
+#include <verbosity.h>
+#include <command.h>
+#include <gfx/video_driver.h>
+#include <wiiu/hid.h>
```

Comment:
I don't like the relative includes either, but I think this will be vetoed.
it is also bad practice to use the <> includes with project specific files, the "" notation will also work.